modules/core/src/main/scala/gem/Semester.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+
+import atto._, Atto._
+import gem.enum.{ Half, Site }
+import java.time._
+import java.time.Month._
+import scalaz.Order, scalaz.syntax.semigroup._
+
+/** A (Year, Half) pair. */
+final case class Semester(year: Year, half: Half) {
+
+  /**
+   * This Semester plus the given number of years.
+   * @throws DateTimeException for final year out of range -999999999 - 999999999
+   */
+  def plusYears(n: Int): Semester =
+    copy(year = year.plusYears(n.toLong))
+
+  /** This Semester plus the given number of half-years. */
+  def plusSemesters(n: Int): Semester = {
+    val yy = year.getValue
+    val hs = yy * 2 + half.toInt * yy.signum + n
```

Comment:
`Semester` takes an arbitrary `Year`, which can be negative. It's just to satisfy the tests really.